pandas-stubs/core/frame.pyi

```diff
@@ -184,7 +198,7 @@ class DataFrame:
     def rename(self, mapper: Callable, axis: _AxisType = ...) -> DataFrame: ...
     @overload
     def rename(self, columns: Dict[_str, _str]) -> DataFrame: ...
-    def replace(self, a: float, b: float) -> DataFrame: ...
+    def replace(self, a: _DTypeEx, b: _DType, regex: bool = ...) -> DataFrame: ...
```

Comment:
Why use a TypeVar here? Just do
```suggestion
    def replace(self, a: Union[_np.dtype, str], b: Union[_np.dtype, str], regex: bool = ...) -> DataFrame: ...
```

TypeVars are only needed if the type of the arguments influence the return type

Reply:
good point